src/lib.rs

```diff
@@ -257,13 +267,6 @@ pub fn new_mnemonic() -> Mnemonic {
     Mnemonic::new(bip39::MnemonicType::Words24, bip39::Language::English)
 }
 
-/// Derives a sr25519::Pair from a Mnemonic
-pub fn mnemonic_to_pair(m: &Mnemonic) -> Result<sr25519::Pair, ValidationErr> {
```

Comment:
This had to be removed as it is no longer possible to use `from_phrase` on wasm with sp-core 21.  But this function was only used in tests and is not exposed to the wasm API.